components/lib/crash/README.md

```diff
@@ -82,13 +82,33 @@ Add a `MozillaSocorroService` instance to your `CrashReporter` in order to uploa
 ```Kotlin
 CrashReporter(
     services = listOf(
-		MozillaSocorroService(applicationContext, "your app name"),
+		MozillaSocorroService(applicationContext, "your app name")
     )
 ).install(applicationContext)
 ```
 
 ⚠️ Note: Currently only native code crashes get uploaded to Socorro. Socorro has limited support for "uncaught exception" crashes too, but it is recommended to use a more elaborate solution like Sentry for that.
 
+### Sending crash reports to Glean
+
+[Glean](https://docs.telemetry.mozilla.org/concepts/glean/glean.html) is a new way to collect telemetry by Mozilla. 
+This will record crash counts as a labelled counter with each label corresponding to a specific type of crash (native_code_crash, and unhandled_exception, currently).
```

Comment:
readability nits: `native_code_crash`, `unhandled_exception` (wrap them in backticks)

Reply:
I'd add a sentence somewhere in here stating "The list of collected metrics is available in the `[metrics.yaml file](metrics.yaml)`" (basically, I'd link to the file.